src/commands/thread.rs

```diff
@@ -0,0 +1,18 @@
+use serenity::all::{ChannelType, CreateThread};
+use crate::{Context, Error};
+
+#[poise::command(slash_command, prefix_command)]
+pub async fn thread(ctx: Context<'_>, thread_name: String) -> Result<(), Error> {
```

Comment:
add annotation for `thread_name` parameter, and split into multiple lines to make it more readable
```suggestion
pub async fn thread(
        ctx: Context<'_>,
        #[description = "thread name"]
        thread_name: String)
-> Result<(), Error> {
```